src/app/modules/main/communities/module.nim

```diff
@@ -105,7 +105,7 @@ method getCommunityItem(self: Module, c: CommunityDto): SectionItem =
           localNickname = contactDetails.details.localNickname,
           alias = contactDetails.details.alias,
           icon = contactDetails.icon,
-          onlineStatus = OnlineStatus.Offline, # TODO get the actual status?
+          onlineStatus = OnlineStatus.Inactive, # TODO get the actual status?
```

Comment:
That's a leftover of me being lazy iirc lol.
I think we can just use `getStatusForContactWithId` from the contactService to get the status of the user.